witchcraft-server/src/logging/logger/async.rs

```diff
@@ -84,9 +84,11 @@ pub struct AsyncAppender<T> {
     state: Arc<Mutex<State<T>>>,
 }
 
-impl<T> Drop for AsyncAppender<T> {
```

Comment:
And restoring this :D

Reply:
Ah shoot, I forgot which one I added and which one I removed. Should have looked at the diff!

Reply:
Should be fixed now!